### PR TITLE
NO SHOES!!! NO SHIRT!!!!

### PR DESCRIPTION
**Pull request by Catdere**
This one is the real one it adds rd clothing centcom, stuff

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Head/hats.yml [hunk 1]
```diff
@@ -761,3 +761,18 @@
...
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatCentcap
+  name: Centcom Commander Cap
+  description: A grand, stylish Commander cap.
+  components:
+  - type: Sprite
+    sprite: Clothing\Head\Hats\centcomcap.rsi
+  - type: Clothing
+    sprite: Clothing\Head\Hats\centcomcap.rsi
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Entities/Clothing/Head/hats.yml]:
```suggestion
    sprite: Clothing/Head/Hats/centcomcap.rsi
```
replace all your stuff with front slashes, NOT blackslashes

File: Resources/Prototypes/Entities/Clothing/Head/hats.yml [hunk 2]
```diff
@@ -761,3 +761,18 @@
...
     - HamsterWearable
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatCentcap
+  name: Centcom Commander Cap
+  description: A grand, stylish Commander cap.
+  components:
+  - type: Sprite
+    sprite: Clothing\Head\Hats\centcomcap.rsi
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Entities/Clothing/Head/hats.yml]:
```suggestion
    sprite: Clothing/Head/Hats/centcomcap.rsi
```

File: Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml [hunk 3]
```diff
@@ -342,3 +342,40 @@
...
         Caustic: 0.75
+
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: ClothingOuterCentcomJacket
+  name: Centcom's Winter Jacket
+  description: A sturdy, utilitarian jacket designed to protect CC when out and about
+  components:
+  - type: Sprite
+    sprite: Clothing\OuterClothing\WinterCoats\centcoms-armoured-wintercoat.rsi
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml]:
```suggestion
    sprite: Clothing/OuterClothing/WinterCoats/centcoms-armoured-wintercoat.rsi
```

File: Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml [hunk 4]
```diff
@@ -342,3 +342,40 @@
...
+  - type: Clothing
+    sprite: Clothing\OuterClothing\WinterCoats\centcoms-armoured-wintercoat.rsi
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml]:
```suggestion
    sprite: Clothing/OuterClothing/WinterCoats/centcoms-armoured-wintercoat.rsi
```

File: Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml [hunk 5]
```diff
@@ -342,3 +342,40 @@
...
+    damageCoefficient: 0.90
+
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: ClothingOuterRDWinterJacket
+  name: RD's Winter Jacket
+  description: Rise... and shine.. Doctor. Freeman. Rise and shine
+  components:
+  - type: Sprite
+    sprite: Clothing\OuterClothing\WinterCoats\rdwinterjacket.rsi
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml]:
```suggestion
    sprite: Clothing/OuterClothing/WinterCoats/rdwinterjacket.rsi
```

File: Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml [hunk 6]
```diff
@@ -342,3 +342,40 @@
...
+  - type: Clothing
+    sprite: Clothing\OuterClothing\WinterCoats\rdwinterjacket.rsi
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml]:
```suggestion
    sprite: Clothing/OuterClothing/WinterCoats/rdwinterjacket.rsi
```

File: Resources/Prototypes/Roles/Jobs/Science/research_director.yml [hunk 7]
```diff
@@ -27,7 +27,7 @@
     jumpsuit: ClothingUniformJumpsuitResearchDirector
     back: ClothingBackpackResearchDirectorFilled
     shoes: ClothingShoesColorBrown
-    outerClothing: ClothingOuterCoatRnd
+    outerClothing: ClothingOuterRDWinterJacket
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Roles/Jobs/Science/research_director.yml:30]:
```suggestion
    outerClothing: ClothingOuterCoatRnd
```
Don't change this

**Review by VasilisThePikachu — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/Head/hats.yml (hunk 1 above)
Comment (Catdere) [reply Resources/Prototypes/Entities/Clothing/Head/hats.yml]: HUH do they really have backslashes am i crazy??

**Review by VasilisThePikachu — APPROVED** [review]

**Pull request merged**